benchmark/src/main/resources/benchmark.xsd

```diff
@@ -458,6 +458,9 @@
           <xs:element minOccurs="0" name="constraintStreamAutomaticNodeSharing" type="xs:boolean"/>
                               
           
+          <xs:element minOccurs="0" name="constraintStreamProfilingMode" type="tns:constraintProfilingMode"/>
```

Comment:
In the interest of not asking the user to make niche choices, I'd remove this. See below.